modules/openapi-generator/src/main/resources/python/python-experimental/model_utils.mustache

```diff
@@ -1024,9 +1024,10 @@ def get_oneof_instance(self, model_args, constant_args):
     schemas described by oneOf.
     Args:
         self: the class we are handling
-        model_args (dict): var_name to var_value
+        model_args (dict/None): var_name to var_value
```

Comment:
With the neswest code we will never reach here when deserializing None.
Can you remove the None additions from this docstirng?
- this also applies to get_anyof_instances